pandas/tests/arrays/datetimes/test_accumulator.py

```diff
@@ -0,0 +1,46 @@
+import pytest
```

Comment:
can you call this test_cumulative.py to match tests/series/ and tests/frame/

Reply:
Done